_episodes/01-welcome.md

```diff
@@ -32,58 +40,63 @@ to conform to our [Code of Conduct]({{ site.coc }}). This Code of Conduct applie
 Hello everyone, and welcome to the the Carpentries
 instructor training.  We're very pleased to have you with us.
 
-> ## Information for Today's Trainers
+> ## Today's Trainers
 >
-> To begin your class, each Trainer should give a brief introduction that will convey (a) their capacity to teach the material, (b) accessibility/approachability, (c) desire for trainee success, and (d) enthusiasm. Tailor your introduction to the learners’ skill level so that you convey competence (without seeming too advanced) and demonstrate that you can relate to the learners. Throughout the workshop, continually demonstrate that you are interested in trainee progress and that you are enthusiastic about the topics. [We will have the opportunity to [work more on this](http://carpentries.github.io/instructor-training/19-introductions/index.html) later on in the course, so don't be daunted by the criteria above!]
+> To begin class, each Trainer should give a brief introduction of themselves.
 {: .discussion}
 
-> ## Information for Today's Learners
->
-> 1. Add your name to the Etherpad
-> 1. Introduce yourselves (preferably verbally)! In your introduction, (a) explain your work in 3 words and (b) say something you are proud of (not necessarily related to research or teaching).
-{: .discussion}
+Now we would like to get to know all of you.
 
-> ## Our First Exercise
+> ## Looking Backwards and Forwards
+>
+> Please answer the following two questions next to your name on the shared notes document:
+> 1. If you have taken part in a Software, Data, or Library Carpentry workshop,
+> how do your experiences compare to those of other people?
+> If you haven't,
+> what questions do you have that they might be able to answer?
+> 1. Why are you taking this course? What goals do you have for today and tomorrow?
 >
```

Comment:
I think it would be good to get experience somehow (to know whether to do a more in-depth Carpentries intro or not). 

It's pretty rough as is though, so I'll think this over and see if I can come up with something better.